ios/Tests/Mocks/OktaOidcMock.swift

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2021-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import XCTest
+import OktaOidc
+@testable import ReactNativeOktaSdkBridge
+
+class OktaOidcMock: OktaOidcProtocol {
+    let configuration: OktaOidcConfig
+    
+    private let shouldFail: Bool
+    
+    // expires in 2037
+    static let mockIdToken = """
+    eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJ3d3cuZXhhbXBsZS5jb20iLCJpYXQiOjE2MTg0NzU1ODMsImV4cCI6MjEyMzM5NzE4MywiYXVkIjoid3d3LmV4YW1wbGUuY29tIiwic3ViIjoianJvY2tldEBleGFtcGxlLmNvbSIsIkdpdmVuTmFtZSI6IkpvaG5ueSIsIlN1cm5hbWUiOiJSb2NrZXQiLCJFbWFpbCI6Impyb2NrZXRAZXhhbXBsZS5jb20ifQ.WY2K4adyY9p--NN83REjZzZglUI7JjxPNvmI3VigGFo
+    """
+    
+    static let mockAccessToken = mockIdToken
+    
+    init(configuration: OktaOidcConfig, shouldFail: Bool) {
+        self.configuration = configuration
+        self.shouldFail = shouldFail
+    }
+    
+    func signInWithBrowser(from presenter: UIViewController,
+                           additionalParameters: [String: String],
+                           callback: @escaping ((OktaOidcStateManager?, Error?) -> Void)) {
+        let oidcManager = Self.oidcStateManager(with: configuration)
+        callback(shouldFail ? nil : oidcManager, shouldFail ? OktaReactNativeError.oktaOidcError : nil)
+    }
+    
+    func signOutOfOkta(_ authStateManager: OktaOidcStateManager, from presenter: UIViewController, callback: @escaping ((Error?) -> Void)) {
+        callback(shouldFail ? OktaReactNativeError.oktaOidcError : nil)
+    }
+    
+    func authenticate(withSessionToken sessionToken: String, callback: @escaping ((OktaOidcStateManager?, Error?) -> Void)) {
+        let oidcManager = Self.oidcStateManager(with: configuration)
+        callback(shouldFail ? nil : oidcManager, shouldFail ? OktaReactNativeError.oktaOidcError : nil)
+    }
+    
+    static func oidcStateManager(with configuration: OktaOidcConfig) -> OktaOidcStateManager {
```

Comment:
Maybe this can be called from the initializer, and the oidcManager can just be a `let` property on the object?  The configuration is being supplied to the initializer, so the manager can be constructed at that time.